packages/react-magma-dom/src/components/Input/InputMessage.tsx

```diff
@@ -15,12 +15,11 @@ export interface InputMessageProps
 
 const Message = styled.div<InputMessageProps>`
   align-items: center;
-  background: ${props =>
-    props.isInverse && props.hasError ? props.theme.colors.danger : 'none'};
+  background: none;
```

Comment:
```suggestion
```

Reply:
I think we can get rid of this line entirely, since the browser default is no background